src/pages/ReimbursementAccount/BusinessInfo/substeps/AddressBusiness.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {withOnyx} from 'react-native-onyx';
+import {View} from 'react-native';
+import useLocalize from '../../../../hooks/useLocalize';
+import styles from '../../../../styles/styles';
+import Text from '../../../../components/Text';
+import CONST from '../../../../CONST';
+import Form from '../../../../components/Form';
+import ONYXKEYS from '../../../../ONYXKEYS';
+import subStepPropTypes from '../../subStepPropTypes';
+import * as ValidationUtils from '../../../../libs/ValidationUtils';
+import {reimbursementAccountPropTypes} from '../../reimbursementAccountPropTypes';
+import getDefaultStateForField from '../../utils/getDefaultStateForField';
+import AddressForm from '../../AddressForm';
+
+const propTypes = {
+    /** Reimbursement account from ONYX */
+    reimbursementAccount: reimbursementAccountPropTypes.isRequired,
```

Comment:
reimbursementAccount shouldnt be required, we need to provide defaultProps for this. Its a change for all components using it

```
import * as ReimbursementAccountProps from '@pages/ReimbursementAccount/reimbursementAccountPropTypes';

const defaultProps = {
    reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
};```